content/docs/command-reference/push.md

```diff
@@ -191,7 +180,6 @@ $ dvc push --with-deps model.p.dvc
 
 $ dvc status --cloud
 
-Data and pipelines are up to date.
 ```
```

Comment:
Oops in https://github.com/iterative/dvc.org/pull/1584#discussion_r459792834 I meant the line above. It was 187 before 168 was removed.

Reply:
Do you mean the current line 182 should be deleted?

Reply:
The empty line... Not `Data and pipelines are up to date.` Sorry, I thought it was obvious...

Reply:
OK I see it's done.